app/src/main/java/com/framgia/soundclound/screen/playlist/PlaylistFragment.java

```diff
@@ -29,9 +33,21 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
                              Bundle savedInstanceState) {
         FragmentPlaylistBinding fragmentPlaylistBinding = DataBindingUtil.inflate(inflater,
                 R.layout.fragment_playlist, container, false);
-        PlaylistModelView playlistModelView = new PlaylistModelView(this.getContext());
-        fragmentPlaylistBinding.setPlaylistModelview(playlistModelView);
+         mPlaylistModelView = new PlaylistModelView(this.getContext());
```

Comment:
ERROR: 'method def' child have incorrect indentation level 9, expected level should be 8.